.circleci/config.yml

```diff
@@ -80,6 +80,10 @@ jobs:
       - run:
           name: "Run Tests - azuresql"
           command: ./run_test.sh azuresql
+      - run:
+          name: "Run Tests - azuresql (retry when server is off)"
+          command: ./run_test.sh azuresql
+          when: on_fail
```

Comment:
Perhaps instead, a single job step:
```
      - run:
          name: "Run Tests - azuresql"
          command: |
              dbt debug --target azuresql
              ./run_test.sh azuresql
```
This should work because:
1. Circle only cares about the exit code returned by the last command run
2. `dbt debug` only returns `0` exit codes, anyway